### ISSUE TITLE
forking

### ISSUE THREAD

**Issue by theBowja** [LichKing112/gi-id-api#1]
Hey I made a lot of changes in [my fork](https://github.com/theBowja/gi-id-api) to the point that the code is pretty much unrecognizable from yours.

I have a lot planned for the [todo](https://github.com/theBowja/gi-id-api/blob/main/todo.md) and was wondering if you wanted to work together on it. If so, I'll make a pull request. Otherwise I figured I may as well start a new repo instead.

**Comment by taksumaki** [comment]
Will you upload it to npmjs? I like your improvements @theBowja

**Comment by taksumaki** [comment]
This package even stopped being in npmjs

**Comment by theBowja** [comment]
@taksumaki I made a new repo at https://github.com/theBowja/genshin-db. It should be in npm under "genshin-db". There's more stuff than in the fork. But I haven't added anything new to the repo recently because I was waiting to see if the genshin api project would release anything new. (And if Genshin's file encryption would be cracked soon so I could get translation files). Unfortunately nothing happened for the past week so I waited for nothing lol.
